src/App.tsx

```diff
@@ -1,61 +1,158 @@
-          Add
-        </button>
-      </form>
-
-      <section className="TodoList">
-        <article data-id="1" className="TodoInfo TodoInfo--completed">
-          <h2 className="TodoInfo__title">
-            delectus aut autem
-          </h2>
-
-          <a className="UserInfo" href="mailto:Sincere@april.biz">
-            Leanne Graham
-          </a>
-        </article>
-
-        <article data-id="15" className="TodoInfo TodoInfo--completed">
-          <h2 className="TodoInfo__title">delectus aut autem</h2>
-
-          <a className="UserInfo" href="mailto:Sincere@april.biz">
-            Leanne Graham
-          </a>
-        </article>
-
-        <article data-id="2" className="TodoInfo">
-          <h2 className="TodoInfo__title">
-            quis ut nam facilis et officia qui
-          </h2>
-
-          <a className="UserInfo" href="mailto:Julianne.OConner@kory.org">
-            Patricia Lebsack
-          </a>
-        </article>
-      </section>
+    <div className="containerForApp">
+      <div className="App">
+        <h1 className="title">
+          Add todo form
+        </h1>
+
+        <form
+          action="/api/users"
+          method="POST"
+          onSubmit={handleSubmit}
+          className="formUsers"
+        >
+          <div className="field">
+            <label
+              htmlFor="titleInput"
+            >
+              Title
+              <input
+                type="text"
+                className="form-field"
+                data-cy="titleInput"
+                value={title}
+                onChange={handleTitle}
+                placeholder="Enter a title"
+              />
+            </label>
+            {errorTitle && (
+              <span className="error">
+                Please enter a title
+              </span>
+            )}
+          </div>
+
+          <div className="field">
+            <label
+              htmlFor="userSelect"
+              className="labelUser"
+            >
+              User
+              <select
+                data-cy="userSelect"
+                id="userSelect"
+                placeholder="Choose a user"
+                className='selectorUsers'
+                value={selectedUserId}
+                onChange={handleUserId}
+              >
+                <option value="0">
+                  Choose a user
+                </option>
+
+                {usersFromServer.map(({ name, id }) => (
+                  <option
+                    value={id}
+                    key={id}
+                  >
+                    {name}
+                  </option>
+                ))}
+              </select>
+            </label>
+
+            {errorUser && (
+              <span className="error">
+                Please choose a user
+              </span>
+            )}
+          </div>
+
+          <button
+            type="submit"
+            data-cy="submitButton"
+            className="button-26"
+          >
```

Comment:
nit: Consider using a more descriptive class name for the submit button, e.g. 'button-submit' instead of 'button-26' 🎨